docs/pages/install/rhel.mdx

```diff
@@ -0,0 +1,96 @@
+# Install Pelican on CentOS, Fedora, or openSUSE
```

Comment:
I'd also include Alma/Rocky here (since they're also supposed to be RHEL variants)